servicemesh-install/topics/tutorial-kiali.adoc

```diff
@@ -38,27 +40,36 @@ image:kiali-login.png[Login Page]
 +
 Log in to the Kiali console using the user name and password that you specified in the custom resource file during installation.
 
+[[kiali-overview-page]]
+== Overview page
+
+After you log in you see the Overview page, which provides you with a quick overview of the health the various namespaces in your system.
+
+image:kiali-overview.png[Overview Page]
+
 [[kiali-graph-page]]
 == Graph page
 
-After you log in you see the Graph page.  This page shows a graph with all the microservices, connected by the requests going through them. On this page, you can see how the services interact with each other.
+. Click the *Graph* link in the left navigation.  The Graph page shows a graph with all the microservices, connected by the requests going through them. On this page, you can see how the services interact with each other.
 
-. From the *Namespace* menu, select Bookinfo.  Now the graph displays only the services in the Bookinfo application.
+. From the *Namespace* menu, select `bookinfo`.  Now the graph displays only the services in the Bookinfo application.
```

Comment:
Are you clicking or selecting? If you're clicking, s/select `bookinfo`/click \*bookinfo\*

Reply:
Please double-check the UI element against the table in the style guide that starts on page 208 and make sure I've got it right. I know it's "click" in more places than it's "select."

Reply:
Bah!  It is select!  "Use select to refer to selecting an item from the list.  Use click when referring to the arrow."   *off to change it back*